keel-bakery-history-sql/keel-bakery-history-sql.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+  `java-library`
+  id("kotlin-spring")
+}
+
+dependencies {
+  implementation(project(":keel-bakery-plugin"))
+  implementation(project(":keel-sql"))
+
+  testImplementation("com.netflix.spinnaker.kork:kork-sql-test")
+  testImplementation("io.strikt:strikt-core")
+  testImplementation(project(":keel-spring-test-support"))
+  testImplementation(project(":keel-test"))
+  testImplementation(project(":keel-core-test"))
+
+  testImplementation(project(":keel-web")) {
+    // avoid circular dependency which breaks Liquibase
+    exclude(module = "keel-bakery-history-sql")
+  }
+}
```

Comment:
New module because I didn't want `keel-sql` to depend on `keel-bakery-plugin`

Reply:
Wondering if I should instead have `keel-bakery-plugin` depend on `keel-sql`. Not sure. I'm not entirely happy with adding this new module.

Reply:
Is it reasonable to keep all sql in one module? or does that present a lot of problems?